src/ert/_c_wrappers/enkf/res_config.py

```diff
@@ -863,7 +863,12 @@ def _config_content_as_dict(config_content: ConfigContent) -> Dict[str, List[Any
         for key in config_content.keys():
             item = config_content[key]
             if key in SINGLE_VALUE_KEYS:
-                content_dict[key] = item.getValue()
+                if key in content_dict:
```

Comment:
This seems really difficult to follow. What is going on?

Reply:
The name SINGLE_VALUE_KEYS led me to believe this section was for parsing keywords that take a single value as argument. But of course it is for parsing keywords that are only allowed to occur once (and also only have one value?). Removed this code.